elements/lisk-p2p/src/p2p.ts

```diff
@@ -531,13 +530,34 @@ export class P2P extends EventEmitter {
 		}
 	}
 
-	public getNetworkStatus(): P2PNetworkStatus {
-		return {
-			newPeers: [...this._peerBook.newPeers],
-			triedPeers: [...this._peerBook.triedPeers],
-			connectedPeers: this._peerPool.getAllConnectedPeerInfos(),
-			connectedUniquePeers: this._peerPool.getUniqueConnectedPeers(),
-		};
+	public getConnectedPeers(): ReadonlyArray<P2PDiscoveredPeerInfo> {
+		return this._peerPool.getAllConnectedPeerInfos();
+	}
+
+	public getUniqueOutboundConnectedPeers(): ReadonlyArray<
```

Comment:
should we modify this to allow for both `outbound` and `inbound`?

Reply:
For now, we don't need `Inbound` due to security reasons. That's why I haven't done it like that.